modules/light-clients/08-wasm/keeper/msg_server_test.go

```diff
@@ -67,11 +67,22 @@ func (suite *KeeperTestSuite) TestMsgStoreCode() {
 			},
 			ibcerrors.ErrUnauthorized,
 		},
+		{
+			"failure: code has could not be pinned",
```

Comment:
Thanks for adding the test case when unpin fails, @DimitrisJim . I have added this one when pin fails in store code.